src/meshapi/tasks.py

```diff
@@ -12,21 +13,61 @@
 )
 from meshapi.views.panoramas import sync_github_panoramas
 from meshdb.celery import app as celery_app
+from meshdb.settings import DBBACKUP_STORAGE_OPTIONS
 
 
 @celery_app.task
-def run_database_backup() -> None:
-    logging.info("Running database backup task")
+def run_database_backup() -> bool:
+    # Don't run a backup unless it's prod1
+    environment = f"\"{os.environ.get('MESHDB_ENVIRONMENT', '')}\""
+    if os.environ.get("MESHDB_ENVIRONMENT") != "prod1":
```

Comment:
We probably should put the env names into a constant somewhere

Reply:
I disagree. This is directly derived from the github action that built the helm chart. It would just make one more step for creating a new environment for no reason. Plus it could be useful for local dev purposes.

Reply:
I'll counter that it's not obvious to a dev what the env names are and an enum listing all of them would make questions like "is `dev` for beta deployments or local dev?" much more obvious. Maybe a wiki page would be good?

Reply:
We should definitely make a wiki page. I'll think about making an enum.

Reply:
Stubbed one out: https://wiki.mesh.nycmesh.net/books/6-services-software/page/environments